### PR TITLE
Add install devices API in spec [API Spec][Application] 

### PR DESCRIPTION
**Pull request by pallavigopi**

### CODE REVIEW

**Review by abdulMuneerM — APPROVED** [review]

**Review by tarun91vas — APPROVED** [review]

**Comment by tarun91vas** [comment]
@tr0n-p46  Can you review this one ?

File: spec/paths/enterprise@{enterprise_id}@application@{application_id}@version@{version_id}@installedevices.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+get:
```
Comment (tr0n-p46) [inline spec/paths/enterprise@{enterprise_id}@application@{application_id}@version@{version_id}@installedevices.yaml:1]: Typo:
...@installdevices.yaml
Test using npm install && npm start

File: spec/definitions/InstallDevices.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+type: object
+properties:
+  id:
+    title: Id
+    type: string
+    format: uuid
+  device_name:
+    title: Device name
+    type: string
+  alias_name:
```
Comment (tr0n-p46) [inline spec/definitions/InstallDevices.yaml:12]: Add nullable

Comment (pallavigopi) [reply spec/definitions/InstallDevices.yaml:12]: @tr0n-p46 are you referring to  device_name or alias_name?

**Review by tr0n-p46 — APPROVED** [review]

**Pull request merged**